hyperspy/io_plugins/bruker.py

```diff
@@ -743,8 +743,8 @@ def _set_elements(self, root):
             for j in elements.findall(
                     "./ClassInstance[@Type='TRTSpectrumRegion']"):
                 tmp_d = dictionarize(j)
-                self.elements[tmp_d['XmlClassName']] = {'line': tmp_d['Line'],
-                                                        'energy': tmp_d['Energy']}
+                self.elements[tmp_d['XmlClassName']] = {'line': tmp_d.get('Line', 'Ka'),
```

Comment:
Add a comment to explain, why we fall back to `'Ka'` when there is no `'line'` key?

Reply:
Thanks for following this up. I added: "In case no information on the specific selected X-ray line is available, assume it is a 'Ka' line, reflecting the fact that element tables in Bruker Esprit (1.9 and 2.1) store a single K line for Li-Al"